distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -160,13 +161,17 @@ type Array
        Arguments:
        - order: The order in which the array elements are sorted.
        - on: A projection from the element type to the value of that element
-         being sorted on.
+         being sorted on. If set to `Nothing` (the default argument),
+         identity function will be used.
        - by: A function that compares the result of applying `on` to two
-         elements, returning an Ordering to compare them.
+         elements, returning an an `Ordering` if the two elements are comparable
+         or `Nothing` if they are not. If set to `Nothing` (the default argument),
+         `Ordering.compare _ _` method will be used.
+       - on_incomparable: A `Problem_Behavior` specifying what should happen if
+         two incomparable values are encountered.
```

Comment:
What happens if values having _the same_ comparator still are not comparable? I assume the warning is still issued, I guess the order is then just undefined? I would like this case to be documented too.

(but btw. very great docs for the _'different comparators'_ case, kudos)

Reply:
Added additional doc section for that